lib/v2/zhihu/posts.js

```diff
@@ -40,7 +40,7 @@ module.exports = async (ctx) => {
             title: articlesdata[key].title,
             description: $.html(),
             link: articlesdata[key].url,
-            pubDate: parseDate(articlesdata[key].created, 'X'),
+            pubDate: parseDate(articlesdata[key].created * 1000),
```

Comment:
There should be no differences if you convert the unix timestamp to unix ms timestamp and specify the time format in the first place
docs: https://day.js.org/docs/en/parse/string-format#list-of-all-available-parsing-tokens